### PR TITLE
Fixing cheveron/plus/minus icon positioning in numberfield buttons

### PR DESCRIPTION
**Pull request by LFDanLu**
Closes 

## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

### CODE REVIEW

File: packages/@adobe/spectrum-css-temp/components/stepper/index.css [hunk 1]
```diff
@@ -139,6 +139,8 @@ governing permissions and limitations under the License.
       inline-size: var(--spectrum-actionbutton-min-width);
       border-block-start-style: solid;
       border-block-start-width: var(--spectrum-stepper-border-size-default);
+      padding-block-end: 0px;
+      padding-block-start: 0px;
```
Comment (LFDanLu) [inline packages/@adobe/spectrum-css-temp/components/stepper/index.css:143]: For +/- icons in large scale

File: packages/@adobe/spectrum-css-temp/components/stepper/index.css [hunk 2]
```diff
@@ -65,10 +65,6 @@ governing permissions and limitations under the License.
 
   padding: var(--spectrum-button-padding-y) calc(var(--spectrum-button-primary-padding-x) - var(--spectrum-button-primary-border-size));
 
-  /* @hack: fix button text vertical alignment at 14px */
-  padding-block-end: calc(var(--spectrum-button-padding-y) + 1px);
-  padding-block-start: calc(var(--spectrum-button-padding-y) - 1px);
```
Comment (LFDanLu) [inline packages/@adobe/spectrum-css-temp/components/stepper/index.css:70]: @snowystinger do you remember what exactly this references? Found the following commit: https://github.com/adobe/react-spectrum/commit/398a051711d90a1231393cee5ffd768488fa1e06, not sure what pointer issue it is referencing

Comment (snowystinger) [reply packages/@adobe/spectrum-css-temp/components/stepper/index.css:70]: o, it was this issue https://github.com/adobe/react-spectrum/pull/1523/files#diff-baa7d39d9b1465a7d8ce5b1760066982303b4f8a93fc797af787879bef41bfb4R94

Comment (snowystinger) [reply packages/@adobe/spectrum-css-temp/components/stepper/index.css:70]: this hack is probably from spectrum-css button code?

Comment (LFDanLu) [reply packages/@adobe/spectrum-css-temp/components/stepper/index.css:70]: gotcha, well it seems to work for me in safari still so hopefully we are fine here

Comment (snowystinger) [reply packages/@adobe/spectrum-css-temp/components/stepper/index.css:70]: yep, it's a div, you won't have a problem functionally

**Comment by snowystinger** [comment]
Mobile looks good, but desktop looks like the chevrons aren't quite right vertically centered, they both seem too close to each other (top is too low, bottom is too high)? is it an optical illusion?
Screen Shot 2021-02-04 at 4 39 22 PM
vs
Screen Shot 2021-02-04 at 4 46 01 PM

**Comment by snowystinger** [comment]
Screen Shot 2021-02-04 at 5 24 02 PM
Light is XD, dark is ours
Looks like our buttons are actually a little too wide, and then only the top chevron is out of place by the .5px as you noted

**Review by snowystinger — APPROVED** [review]
Looks great on iOS Safari, Android Chrome/Samsung/FF browsers, MacOS Chrome/Safari/FF

**Review by devongovett — APPROVED** [review]

**Pull request merged**